src/bin/pg_rewind/pg_rewind.c

```diff
@@ -129,7 +131,6 @@ main(int argc, char **argv)
 	};
 	int			option_index;
 	int			c;
-	XLogRecPtr	divergerec;
```

Comment:
Last common internal LSN representation (byte position in WALs) needs to be set as a global variable in order to be accessed by `decide_wal_file_action(...)`, as we cannot just pass it into `decide_file_actions(...)`.